admin/modules/user/users.php

```diff
@@ -3596,7 +3596,7 @@ function build_users_view($view)
 			}
 			if(!$user['avatar'])
 			{
-				$user['avatar'] = "../".$mybb->settings['useravatar'];
+				$user['avatar'] = "../".str_replace('{theme}', 'images', $mybb->settings['useravatar']);
```

Comment:
{theme} here isn't actually the theme, it's `images`. I'm sure that isn't intended as per title